packages/radix-ui-themes/src/components/progress.tsx

```diff
@@ -13,9 +13,7 @@ type ProgressOwnProps = GetPropDefTypes<typeof progressPropDefs>;
 interface ProgressProps
   extends Omit<PropsWithoutRefOrColor<typeof ProgressPrimitive.Root>, 'children'>,
     MarginProps,
-    ProgressOwnProps {
-  duration?: string;
-}
```

Comment:
Should we do specify the duration type so that `${number}s` is enforced?

Reply:
We can keep it as 'string' in the docs

Reply:
you mean, override the type in the component?

Reply:
Yeah potentially